server/routes/summary_routes.ts

```diff
@@ -47,13 +47,24 @@ export function registerSummaryAssistantRoutes(
         req.body.index && req.body.dsl && req.body.topNLogPatternData
           ? LOG_PATTERN_SUMMARY_AGENT_CONFIG_ID
           : SUMMARY_AGENT_CONFIG_ID;
-      const response = await assistantClient.executeAgentByConfigName(agentConfigId, {
-        context: req.body.context,
-        question: req.body.question,
-        index: req.body.index,
-        input: req.body.dsl,
-        topNLogPatternData: req.body.topNLogPatternData,
-      });
+
+      let response;
+      try {
+        response = await assistantClient.executeAgentByConfigName(agentConfigId, {
```

Comment:
should this be res instead of response?

Reply:
`response` is the http response body of the backend API, will be consumed below.